PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -2056,13 +2134,13 @@ public void TransparentEffectClick(Office.IRibbonControl control)
             TransparentEffect(selection.ShapeRange);
         }
 
-        public void EffectsLabSettingsButtonClick(Office.IRibbonControl control)
+        /*public void EffectsLabSettingsButtonClick(Office.IRibbonControl control)
         {
             var effectsLabSettingsDialog = new EffectsLabSettings(_effectsLabSelectionIsCover);
             effectsLabSettingsDialog.SettingsHandler += EffectsLabSettingChangedHandler;
             effectsLabSettingsDialog.ShowDialog();
             effectsLabSettingsDialog.Dispose();
-        }
+        }*/

```

Comment:
I was keeping it there so that it would be easier to reactivate the effects settings if needed.